x-pack/plugins/cases/common/ui/types.ts

```diff
@@ -155,8 +155,6 @@ export type UpdateKey = keyof Pick<
 export interface UpdateByKey {
   updateKey: UpdateKey;
   updateValue: CasePatchRequest[UpdateKey];
-  fetchCaseUserActions?: (caseId: string, caseConnectorId: string) => void;
```

Comment:
the main focus on this PR is removing the usage of these props that are passed all the way down from the case view page.